pkg/payload/precondition/clusterversion/upgradeable_test.go

```diff
@@ -60,50 +22,98 @@ func TestUpgradeableRun(t *testing.T) {
 	}
 
 	tests := []struct {
-		name           string
-		upgradeable    *configv1.ConditionStatus
-		currVersion    string
-		desiredVersion string
-		expected       string
+		name               string
+		upgradeable        *configv1.ConditionStatus
+		currVersion        string
+		desiredVersion     string
+		NonBlockingWarning bool
```

Comment:
nit: everything else opens lowercase.  Can we use `nonBlockingWarning` here for consistency?